src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
@drarmstr I don't have a clean approach here. I would say potentially create a map at the Crossfilter scope level with the dimension object as the key and store the offsets there as well as in the dimension scope. I believe they are just defined once when the dimension is created, so you would not have to worry about them changing. The in the isElementFiltered function you could use that map to get the necessary information.